do-catalog/src/App.vue

```diff
@@ -15,6 +15,10 @@ export default {
 };
 </script>
 
-<style lang="scss">
-@import './styles/main.scss';
+<style lang="scss" scoped>
+/* stylelint-disable */
+::v-deep {
+  @import './styles/main.scss';
```

Comment:
It is needed to import external fonts from outside of this selector in order to get fonts working.